packages/inter-protocol/test/smartWallet/test-psm-integration.js

```diff
@@ -438,67 +442,86 @@ test('agoricName invitation source errors', async t => {
   );
   const computedState = coalesceUpdates(E(wallet).getUpdatesSubscriber());
 
-  await wallet.getOffersFacet().executeOffer({
-    id: 'missing property',
-    // @ts-expect-error intentional violation
-    invitationSpec: {
-      source: 'agoricContract',
-      instancePath: ['psm-IST-AUSD'],
-      // callPipe: [['makeGiveMintedInvitation']],
+  await t.throwsAsync(
+    wallet.getOffersFacet().executeOffer({
+      id: 'missing property',
+      // @ts-expect-error intentional violation
+      invitationSpec: {
+        source: 'agoricContract',
+        instancePath: ['psm-IST-AUSD'],
+        // callPipe: [['makeGiveMintedInvitation']],
+      },
+      proposal: {},
+    }),
+    {
+      message:
+        '{"source":"agoricContract","instancePath":["psm-IST-AUSD"]} - Must have missing properties ["callPipe"]',
     },
-    proposal: {},
-  });
+  );
   t.is(await E.get(getBalanceFor(anchor.brand)).value, 0n);
   t.is(await E.get(getBalanceFor(mintedBrand)).value, 0n);
   t.like(computedState.offerStatuses.get('missing property'), {
     error:
       'Error: {"source":"agoricContract","instancePath":["psm-IST-AUSD"]} - Must have missing properties ["callPipe"]',
   });
 
-  await wallet.getOffersFacet().executeOffer({
-    id: 'bad namepath',
-    invitationSpec: {
-      source: 'agoricContract',
-      instancePath: ['not-present'],
-      callPipe: [['makeGiveMintedInvitation']],
-    },
-    proposal: {},
-  });
+  await t.throwsAsync(
+    wallet.getOffersFacet().executeOffer({
+      id: 'bad namepath',
+      invitationSpec: {
+        source: 'agoricContract',
+        instancePath: ['not-present'],
+        callPipe: [['makeGiveMintedInvitation']],
+      },
+      proposal: {},
+    }),
+    { message: '"nameKey" not found: "not-present"' },
```

Comment:
`nameKey` here should be `instance`, I think. (orthogonal to this PR)

Reply:
I think it comes from NameHub https://github.com/Agoric/agoric-sdk/blob/master/packages/vats/src/nameHub.js#L23